### ISSUE TITLE
Any ideas on how to set the right permission on the sites/default/files folder and files?

### ISSUE THREAD

**Issue by timbrandin** [mybethel/docker-drupal-kitematic#1]
Great docker btw!!

**Comment by Ignigena** [comment]
Hey there! Your latest pull request actually solves this problem if I'm not mistaken?

**Comment by timbrandin** [comment]
Yep! :) Btw, do you know of a way to replace the standard folder syncing with NFS? Need to speed up my heavy drupal sites, it's way slower than a regular regular vagrant+vbox setup with NFS atm.
